charts/ingress-nginx/ingress-nginx/values.yaml.gotmpl

```
@@ -25,6 +25,9 @@ controller:
       service.beta.kubernetes.io/aws-load-balancer-backend-protocol: tcp
       service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: 'true'
       service.beta.kubernetes.io/aws-load-balancer-type: nlb
+  {{- if eq .Values.jxRequirements.cluster.provider "aks" }}
```

This needs to be `else if` since you are in the `{{- if eq .Values.jxRequirements.cluster.provider "eks" }}` block
```suggestion
  {{- else if eq .Values.jxRequirements.cluster.provider "aks" }}
```

---

Whoops, apologies